packages/labs/cli/src/lib/init/element-starter/templates/lib/element.ts

```diff
@@ -0,0 +1,164 @@
+import {FileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
+import {
+  javascript,
+  kabobToPascalCase,
+} from '@lit-labs/gen-utils/lib/str-utils.js';
+import {Language} from '../../../../commands/init.js';
+
+export const generateElement = (
+  elementName: string,
+  language: Language
+): FileTree => {
+  const directory = language === 'js' ? 'lib' : 'src';
+  return {
+    [directory]: {
+      [`${elementName}.${language}`]:
+        language === 'js' ? js(elementName) : ts(elementName),
+    },
+  };
+};
+
+const js = (elementName: string) => {
+  const className = kabobToPascalCase(elementName);
+  return javascript`/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import { LitElement, html, css } from 'lit';
+
+/**
+ * An example element.
+ *
+ * @fires count-changed - Indicates when the count changes
+ * @slot - This element has a slot
+ * @csspart button - The button
+ */
+export class ${className} extends LitElement {
+  static get styles() {
+    return css\`
+      :host {
+        display: block;
+        border: solid 1px gray;
+        padding: 16px;
+        max-width: 800px;
```

Comment:
Suggest removing the `max-width`. Doesn't seem like that's an obvious "include in the default.."